filename/filename.go

```diff
@@ -40,18 +45,22 @@ func NewHook(levels ...logrus.Level) *Hook {
 	return &hook
 }
 
-func findCaller(skip int) (string, string, int) {
+func findCaller(skip int, prefixes []string) (string, string, int) {
 	var (
 		pc       uintptr
 		file     string
 		function string
 		line     int
 	)
+frame_loop:
 	for i := 0; i < 10; i++ {
 		pc, file, line = getCaller(skip + i)
-		if !strings.HasPrefix(file, "logrus/") {
-			break
+		for _, prefix := range prefixes {
```

Comment:
I think will be better to user map here.

Reply:
Use how? Do you mean change `Prefixes []string` to `Prefixes map[string]struct{}` and use it as a set?

Reply:
Done.